itest/litd_custom_channels_test.go

```diff
@@ -1972,18 +1972,6 @@ func testCustomChannelsLiquidityEdgeCases(ctx context.Context,
 	logBalance(t.t, nodes, assetID, "after big asset payment (asset "+
 		"invoice, multi-hop)")
 
-	// Edge case: Now Charlie creates a tiny asset invoice to be paid for by
-	// Yara with satoshi. This is a multi-hop payment going over 2 asset
-	// channels, where the total asset value is less than the default anchor
-	// amount of 354 sats.
-	createAssetInvoice(t.t, dave, charlie, 1, assetID, withInvoiceErrSubStr(
```

Comment:
Why are we removing this edge case test? This should still apply IMO. The error message has changed a bit though.